src/Network/Network/help/New-AzFirewallPolicyApplicationRule.md

```diff
@@ -38,6 +38,15 @@ PS C:\> New-AzFirewallPolicyApplicationRule -Name AR1 -SourceAddress "192.168.0.
 
 This example creates an application rule with the source address, protocol and the target fqdns.
 
+### Example 2
+
+Create a new Azure Firewall Policy Application Rule. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+New-AzFirewallPolicyApplicationRule -Description 'Allow RDP' -Name AR1 -Protocol 'http:80' -SourceAddress '192.168.0.0/16' -TargetFqdn '*.ro'
```

Comment:
The value for description or protocol seems inaccurate. RDP defaults to port 3389. Protocol specifies http:80 with is used by non-SSL web servers.